api/controllers/cartController.js

```diff
@@ -0,0 +1,49 @@
+const cartService = require("../services/cartService");
+
+async function insertCartItems(req, res) {
```

Comment:
```suggestion
async function insertCartItem(req, res) {
```
product가 하나인 것 같으니 Items가 아니라 단수형으로 작성하는 것이 좋아보입니다.